cpu/esp_common/freertos/task.c

```diff
@@ -145,6 +145,11 @@ void vTaskSuspend(TaskHandle_t xTaskToSuspend)
     }
 }
 
+void vTaskSuspendAll(void)
+{
+    /* TODO */
```

Comment:
I wonder if we will ever be able to implement it in on top of the RIOT scheduler.